### ISSUE TITLE
Project setup: help needed throughout

### ISSUE THREAD

**Issue by romw** [BOINC/boinc-ng#1132]
**Reported by mm689 on 23 Dec 41921465 19:17 UTC**
As a Computer Science undergraduate assigned the task of setting up a BOINC project for my supervisor's research, I have been attempting to use the BOINC server documentation contained on this wiki. However, there does not appear to be any central guide to setting up a project and, in the pages that do exist, many features are referenced in passing without any technically relevant details being offered, and some documentation is simply incorrect according to what I have been able to glean.

I do not have a specific problem: instead, I am unclear on how to get my server - which is set up with the BOINC preconfigured server running in a VMWare virtual machine - past the initial setup and communicating with clients. I have managed to set it up to be accessible to clients, but have no idea how to set up jobs and versions to actually be distributed.

Any help would be greatly appreciated.

**Comment by romw** [comment]
**Commented by davea on 27 Aug 41921611 12:43 UTC**
Please be more specific about documentation problems.

Please post to boinc_projects@ssl.berkeley.edu for general help w/ the server software.

**Comment by romw** [comment]
**Commented by mm689 on 5 Sep 41921815 04:43 UTC**
I'm unable to post to boinc_projects@ without subscribing to it. How exactly would I do that? Anyway, for now, my issues are as follows.

I have set up the provided virtual machine and am running it in VMWare on a dedicated server with a unique IP etc., such that I can connect to it (http://boinc.cs.st-andrews.ac.uk/minion) from a computer running the client software and successfully connect to my manually-created project. However, once this is done I am at a loss as to what to do next. The sample program, "uppercase", has a single program in its "apps" directory, with no version or job details. My questions at this stage are as follows:
- Where do I define the platforms my program will work on, and how exactly do they link with the client program?
- How do I set up the configuration for a wrapper round my own executables?
- A related side note: the wiki page AppVersionNew clearly instructs to create a hierarchy involving, for example, PROJECT/apps/app_a/1.0/windows_intelx86/version.xml. However, the structure of "uppercase", and restrictions active within the update_versions script, indicate that the directory hierarchy must contain PROJECT/apps/app_a/app_a_1.0/windows_intelx86/app_a_1.0/windows_intelx86. What does the bottom-level program actually do, where is it defined, and most importantly how can I configure and/or replace it?

**Comment by romw** [comment]
**Commented by davea on 30 Jun 41930205 22:27 UTC**
To subscribe to an email list,
go to the BOINC web site and click on "Email lists".

Your three questions are (I believe) answered in
http://boinc.berkeley.edu/trac/wiki/AppVersionNew.

**Comment by romw** [comment]
**Commented by mm689 on 21 Jul 41974212 19:21 UTC**
I emailed boinc_projects a week ago but haven't yet heard back. My issue is simple, at the moment: in my client's event log I'm getting the error "Scheduler request failed: couldn't resolve host name". As my master URL definitely works (that's where I got the project from, and get_project_config.php was found), is this a missed configuration parameter or a file/feature I haven't created? I have of course run bin/start, but should I be running anything else too?
